test/test_logutil.py

```diff
@@ -19,6 +18,10 @@ def emit(self, record):
 class TestLogutil(fixture.SubManFixture):
     def setUp(self):
         super(TestLogutil, self).setUp()
+        self.rhsm_config = stubs.StubConfig()
+        self.rhsm_patcher = mock.patch('rhsm.config')
+        self.rhsm_config_mock = self.rhsm_patcher.start()
```

Comment:
No need to make the actual patcher part of self.  However you need to stop the patcher to avoid polluting other tests: `self.addCleanup(rhsm_patcher.stop)`